vscode/src/rubyLsp.ts

```diff
@@ -394,6 +394,25 @@ export class RubyLsp {
           }
         },
       ),
+      vscode.commands.registerCommand(
+        Command.BundleInstall,
+        (workspaceUri: string) => {
+          const workspace = this.workspaces.get(workspaceUri);
+
+          if (!workspace) {
+            return;
+          }
+
+          const terminal = vscode.window.createTerminal({
+            name: "Bundle install",
+            cwd: workspace.workspaceFolder.uri.fsPath,
+            env: workspace.ruby.env,
+          });
+
+          terminal.show();
+          terminal.sendText("bundle install");
```

Comment:
Is it possible to clear any text the user may have typed before sending this? (I sometimes get failures for the terminal test runner because of an incomplete command I had previously typed).

Reply:
We are always creating a brand new terminal in the lines above, so there's no way you could type anything before running the command.